tests/e2e/project/issues.spec.ts

```diff
@@ -48,7 +48,7 @@ test("test issue editing failing", async ({ page, authenticatedPeer }) => {
   );
 
   await page.goto(
-    `/seeds/127.0.0.1:8070/${rid}/issues/d316f7a90a40dacbfb8728044bad50c9f71d44ba`,
+    `/seeds/${authenticatedPeer.httpdBaseUrl.hostname}:${authenticatedPeer.httpdBaseUrl.port}/${rid}/issues/d316f7a90a40dacbfb8728044bad50c9f71d44ba`,
```

Comment:
I think you can used `authenticatedPeer.uiUrl()` here as well.

Reply:
nice, changed it 👍